src/Aspire.Dashboard/Utils/StringComparers.cs

```diff
@@ -1,19 +1,19 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-namespace Aspire.Dashboard.Utils;
+namespace Aspire;
```

Comment:
Can you move this to shared code directory: https://github.com/dotnet/aspire/tree/main/src/Shared

Referencing files from other projects can create unexpected results.

Reply:
I noticed there were other places this happens. I created an issue: https://github.com/dotnet/aspire/issues/1711